scripts/run_diff.sh

```diff
@@ -76,17 +76,22 @@ git_start=`git merge-base $git_start $git_merge_base_end`
 
 cleanup()
 {
-  rm -f $diff_file
+  rm -f $diff_file $added_lines_file
 }
 
 trap cleanup EXIT
 
 diff_file=`mktemp`
+added_lines_file=`mktemp`
 
-git diff $git_start $git_end > $diff_file
+# Pass the output through iconv to remove any invalid UTF-8 (diff_to_added_lines.py will die otherwise)
 
-# Get the list of files that have changed
-diff_files=`git diff --name-only $git_start $git_end`
+git diff $git_start $git_end | iconv -t utf-8 -c > $diff_file
```

Comment:
`iconv` appears to be widely installed, but you might consider adding a check that it does with an easy to read message that it needs to be installed.